src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -686,6 +715,16 @@ export class BrowsingContextImpl {
     }
   }
 
+  async handleUserPrompt(
+    params: BrowsingContext.HandleUserPromptParameters
+  ): Promise<void> {
+    await this.#cdpTarget.cdpClient.sendCommand('Page.handleJavaScriptDialog', {
+      accept: params.accept ?? true,
+      promptText: params.userText,
+    });
+    return;
```

Comment:
`return {result: {}}`;

Reply:
Not correct we to this in other functions as well when no result is expected we do `void` and the `process_browsingContext_handleUserPrompt` handles the type EmptyResult.